docs/en/index.md

```diff
@@ -6,109 +6,109 @@
 
-Click on the link, then choose "Create an account" in the upper right-hand
-corner.
-<img src="images/SlackAAW2.PNG" alt="A screenshot of the next page, with a box to use your @statcan.gc.ca email" width="50%">
+Built on the Kubeflow project, the AAW is a comprehensive solution for deploying and managing end-to-end ML workflows. It simplifies the deployment of ML workflows on Kubernetes, making it simple, portable, and scalable. With the AAW, you can customize notebook server deployments to suit your specific data science needs. We have a small number of expertly crafted Docker images made by our team of data science experts.
 
-Use your @statcan.gc.ca email address so that you will be automatically approved.
+!!! info "What is Kubeflow?"
+    The AAW is based on the Kubeflow project which is an open source comprehensive solution for deploying and managing end-to-end ML workflows. Kubeflow is designed to make deployments of ML workflows on Kubernetes simple, portable and scalable.
 
-# Experiments
+Whether you're just getting started or already knee-deep in data analysis, the Advanced Analytics Workspace has everything you need to take your work to the next level. From powerful tools for data pipelines to cloud storage for your datasets, our platform has it all. Need to collaborate with colleagues or publish your results? No problem. We offer seamless collaboration features that make it easy to work together and share your work with others.
 
-## Process data using `R`, `Python`, or `Julia`
+No matter what stage of your data science journey you're at, the Advanced Analytics Workspace has the resources you need to succeed.
 
-[![R, Python, or Julia in Jupyter notebooks](images/Jupyter.PNG)](1-Experiments/Jupyter.md)
-Once you have Kubeflow set up, use
-**[Jupyter Notebooks](1-Experiments/Jupyter.md)** to create and share documents
-that contain live code, equations, or visualizations.
-![Jupyter Notebooks](images/jupyter_in_action.png)
+## Getting Started with the AAW
 
-## Process data using 'R' or 'Python'
+<center>
+![image](https://user-images.githubusercontent.com/8212170/158243976-0ee25082-f3dc-4724-b8c3-1430c7f2a461.png)
+</center>
+
+### The AAW Portal
+
+The AAW portal homepage is available for internal users only. However, external users with a cloud account granted access by the business sponsor can access the platform through the analytics-platform URL.
 
-[![R or Python in R Studio](images/RStudio.PNG)](1-Experiments/RStudio.md)
-**[R Studio](1-Experiments/RStudio.md)** gives you an integrated development
-environment for R and Python. Use the r-studio-cpu image to get an R Studio
-environment.
+!!! info annotate "AAW Portal Homepage"
+    - [**Portal Homepage for Statistics Canada Employees**](https://www.statcan.gc.ca/data-analytics-service/aaw)
+    - [**Portal Homepage for External Users**](https://analytics-platform.statcan.gc.ca/covid19)
 
-## Run a virtual desktop
+!!! info "Kubeflow Dashboard"
```

Comment:
Move under kubeflow account section, since it makes chronological sense (they can't skip the account section and be confused)